tvm-tac/src/main/kotlin/org/ton/tac/TacInst.kt

```diff
@@ -0,0 +1,30 @@
+package org.ton.tac
+
+sealed class TacInst(
+    open val mnemonic: String
+)
+
+data class StackTacInst(
+    override val mnemonic: String,
+    val stackState: String,
+    val operands:  Map<String, Any?>
+): TacInst(mnemonic)
+
+data class NonStackTacInst(
+    override val mnemonic: String,
+    val inputs: List<TacVar>,
+    val outputs: List<TacVar>,
+    val operands: MutableMap<String, Any?>, // (?) to be deleted; actually important for 'LDU' for example
+    val continuationsRefs: List<ContinuationRef>? = null,  // if CONT was not pushed on stack i.e. it's in operands like in 'CALLREF'
```

Comment:
`operands` should not be deleted.

What I meant was: instruction input is `input variables` + `operands`. Some input we should treat not like an ordinary input, but as code blocks that control flow can be passed to. That should be moved separately and then printed differently.

For example, instruction `IF` has no operands and two stack inputs. Now it would be printed like this:
```
IF(n_x, c_y)
```

Instead, it should be printed like this:
```
IF(n_x) {
   // disassembled c_y with substitued arguments
}
```